apps/widget-configurator/src/web3modalConfig.ts

```diff
@@ -0,0 +1,47 @@
+import { CHAIN_INFO, RPC_URLS } from '@cowprotocol/common-const'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { createWeb3Modal, defaultConfig } from '@web3modal/ethers5/react'
+
+const WC_PROJECT_ID = process.env.REACT_APP_WC_PROJECT_ID
+const WC_DEFAULT_PROJECT_ID = 'a6cc11517a10f6f12953fd67b1eb67e7'
+
+const projectId = WC_PROJECT_ID || WC_DEFAULT_PROJECT_ID
+
+const metadata = {
+  name: 'CoW Widget Configurator',
+  description: 'Injectable UI of CoWSwap',
+  url: 'https://swap.cow.fi',
+  icons: ['https://swap.cow.fi/favicon.png'],
+}
+
+const ethersConfig = defaultConfig({
+  metadata,
+
+  /*Optional*/
+  enableEIP6963: true,
+  enableInjected: true,
+  enableCoinbase: true, // true by default
+  rpcUrl: '...', // used for the Coinbase SDK
```

Comment:
Don't we need to set this, give that coinbase is enabled?

Reply:
Fixed